src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1087,6 +1093,64 @@ public Inventory get(final InventoryClickEvent e) {
 				return e.getClickedInventory();
 			}
 		}, 0);
+		// InventoryDragEvent
+		EventValues.registerEventValue(InventoryDragEvent.class, Player.class, new Getter<Player, InventoryDragEvent>() {
+			@Override
+			public @Nullable Player get(InventoryDragEvent event) {
+				return event.getWhoClicked() instanceof Player ? (Player) event.getWhoClicked() : null;
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, World.class, new Getter<World, InventoryDragEvent>() {
+			@Override
+			public @Nullable World get(InventoryDragEvent event) {
+				return event.getWhoClicked().getWorld();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, ItemStack[].class, new Getter<ItemStack[], InventoryDragEvent>() {
```

Comment:
Is there is use for [getOldCursor](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/event/inventory/InventoryDragEvent.html#getOldCursor()) Could it be the past timestate of event-itemstack?